samples/subsys/fs/fat_fs/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "FAT Filesystem Sample Application"
+
+config SAMPLE_FATFS_CREATE_SOME_ENTRIES
```

Comment:
Why not make this Kconfig `default y`, or even hardcode the sample to create files when none exist? I think this functionality is useful beyond just QSPI, I often test the fat_fs sample with a freshly formatted SD card, and creating a file would be useful behavior.

Reply:
I am fine with that.